### ISSUE TITLE
Make name of "More actions" button more accessible

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8544]
## Area or Component

Results page - "More actions" button

## Quick Summary

The "More actions" button name is the same for every row of results making it unclear which result this is for.

## Accessibility Violation

[aria-name-missing-incorrect](https://docs.deque.com/issue-help/1.0.0/en/aria-name-missing-incorrect)

## Action Requested

Update the `<span>` with the `usa-sr-only` class to be more descriptive like [we do here](https://github.com/CDCgov/prime-simplereport/blob/f443baacffeea64d435fdba29fd545dacc89de2e/frontend/src/app/testResults/viewResults/ResultsTable.tsx#L157)

## Screenshots (optional)

<img width="1250" alt="Image" src="https://github.com/user-attachments/assets/92a0816b-c017-4b8e-896c-e39a4da45cec" />

## Source Code (optional)

https://github.com/CDCgov/prime-simplereport/blob/f443baacffeea64d435fdba29fd545dacc89de2e/frontend/src/app/commonComponents/ActionsMenu.tsx#L27
